Assets/MixedRealityToolkit-SDK/Features/UX/Scripts/Tooltips/ToolTip.cs

```diff
@@ -0,0 +1,540 @@
+﻿//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+//
+using System;
+using UnityEngine;
+using Microsoft.MixedReality.Toolkit.InputSystem;
+using Microsoft.MixedReality.Toolkit.Core.Extensions;
+using Microsoft.MixedReality.Toolkit.Core.Definitions.Lines;
+using Microsoft.MixedReality.Toolkit.Core.Utilities.Lines.DataProviders;
+
+namespace Microsoft.MixedReality.Toolkit.SDK.UX.ToolTips
+{
+    [RequireComponent(typeof(ToolTipConnector))]
+
+    /// <summary>
+    /// Class for Tooltip object
+    /// Creates a floating tooltip that is attached to an object and moves to stay in view as object rotates with respect to the view.
+    /// </summary>
+    public class ToolTip : MonoBehaviour//, IMixedRealityInputHandler
```

Comment:
why is the input handler interface commented out? is it going to be needed in the future?

Reply:
Resolved in my PR I opened on his branch.